docs/releases.rst

```diff
@@ -0,0 +1,10 @@
+Releases
+========
+
+This page details the changes in the various ``django-formtools`` releases.
+
+1.0
+---
+
+- Added the ``request`` parameter to :meth:`WizardView.get_prefix
```

Comment:
to be super nutpicky: `WizardView.get_prefix()`. Otherwise, this looks good.

By the way, see https://github.com/django/django/commit/6302893112a67343b3f5eff3f09f91b245346877#diff-ea4f460f2becc51cd39d7587ec3f3244L128 for a change made after 1.7 that should be added to the releases notes.